### ISSUE TITLE
Can't pass box requirements checker with PHP 8 and takeout v1.6.5

### ISSUE THREAD

**Issue by vtalbot** [tighten/takeout#191]
Since I upgraded my PHP to 8.0, I get this error whenever I run a `takeout` command:

```
Box Requirements Checker
========================

> Using PHP 8.0.0
> PHP is using the following php.ini file:
  /usr/local/etc/php/8.0/php.ini

> Checking Box requirements:
  E..


 [ERROR] Your system is not ready to run the application.


Fix the following mandatory requirements:
=========================================

 * The application requires the version "^7.2.5" or greater.
```

I'm using the v1.6.5 of `takeout` and I'm on Big Sur

**Comment by mattstauffer** [comment]
@vtalbot Thanks! We'll need to test it out on PHP 8 quickly before we tag support. Are you up to be a guinea pig?

**Comment by mattstauffer** [comment]
Actually.. weird. We already tagged 8.0 support in v1.6.5. I'm not sure why you're getting that error. I'll try to take some time to set up PHP 8 locally and test it out.

**Comment by mattstauffer** [comment]
<img width="1099" alt="image" src="https://user-images.githubusercontent.com/151829/100794835-202c3a80-33ec-11eb-8d66-74878741852f.png">

Same issue here.

Takeout works fine, but once it's boxed up, it doesn't work fine. Let me look into this!

**Comment by mattstauffer** [comment]
Ah.. Laravel Zero 7+ doesn't work with PHP 8. We'll have to update to Laravel Zero 8. Working on that now.

**Comment by mattstauffer** [comment]
And... it doesn't support PHP 8 either.

https://github.com/laravel-zero/framework/pull/421

So we updated *our* tool to support PHP 8, but our dependency doesn't support it, so we jumped the gun on the 8.0 support. :/ Sorry!

**Comment by mattstauffer** [comment]
This should be fixed in v1.8.0. Please test it out. Thanks!
